server/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1408,4 +1413,10 @@ public void test_prohibit_using_aggregations_in_generated_columns() throws Excep
             Matchers.startsWith("Aggregation functions are not allowed in generated columns: max(x)")
         );
     }
+
+    @Test
+    public void test_number_of_routing_shard_validation_is_based_on_number_of_shards_value() throws Exception {
+        // just ensure that this doesn't raise an error.
+        analyze("create table tbl (x int) clustered into 2 shards with (number_of_routing_shards = 1024)");
```

Comment:
Does this test anything additional to the `testNumberOfRoutingShardsCanBeSetAtCreateTable` test?

Reply:
No not really, I guess it could be removed again. I added this before I had noticed that we had tests that relied on the incorrect validation.

Reply:
Removed it